src/var/isAttached.js

```diff
@@ -0,0 +1,8 @@
+define( function() {
+	"use strict";
+
+	return function isAttached( obj ) {
+		return jQuery.contains( obj.ownerDocument, obj );
```

Comment:
This `jQuery.contains` dependency should be captured in the `define` arguments, as it was in the other source files.